quixstreams/dataframe/dataframe.py

```diff
@@ -1512,19 +1585,23 @@ def _groupby_key(
 
     def __dataframe_clone__(
         self,
+        *topics: Topic,
         stream: Optional[Stream] = None,
-        topic: Optional[Topic] = None,
+        stream_id: Optional[str] = None,
     ) -> Self:
         """
-        Clone the StreamingDataFrame with a new `stream` and `topic` parameters.
+        Clone the StreamingDataFrame with a new `stream`, `topics`,
+        and optional `stream_id` parameters.
 
+        :param topics: one or more `Topic` objects
         :param stream: instance of `Stream`, optional.
-        :param topic: instance of `Topic`, optional.
+        :param stream_id: an identifier of the data stream, optional
         :return: a new `StreamingDataFrame`.
         """
+
         clone = self.__class__(
+            *topics or self._topics,
```

Comment:
This would make it slightly more clear
```suggestion
            *(topics or self._topics),
```